scripts/logbook.py

```diff
@@ -38,76 +38,579 @@
-        "Size Selection (Robocut)" : {"dest_file" : ["Bravo"], "instrument" : ["lims_instrument"], "details" : [""]},
-        "Shear DNA (SS XT) 4.0" : {"dest_file" : ["Covaris","Bravo"], "instrument" : ["lims_instrument","udf_Instrument Used"], "details" : [{"udf_Lot no: Covaris tube" : ""},""]},
-        "ThruPlex library amplification" : {"dest_file" : ["Bravo","PCR"], "instrument" : ["lims_instrument", "udf_PCR Machine"], "details" : ["", ""]},
-        "ThruPlex template preparation and synthesis" : {"dest_file" : ["Bravo","PCR"], "instrument" : ["lims_instrument", "udf_PCR Machine"], "details" : ["", ""]}
+        "Adapter ligation and reverse transcription (TruSeq small RNA) 1.0" : {
+            "lims_instrument" : {
+                "dest_file" : "Bravo",
+                "details" : []
+            }
+        },
+        "Adapter Ligation and 1st Amplification (SMARTer Pico) 4.0" : {
+            "lims_instrument" : {
+                "dest_file" : "Bravo",
+                "details" : []
+            },
+            "udf_PCR Machine" : {
+                "dest_file" : "PCR",
+                "details" : []
+            }
+        },
+        "Aliquot Samples for Caliper/Bioanalyzer" : {
+            "lims_instrument" : {
+                "dest_file" : "Bravo",
+                "details" : []
+            }
+        },
+        "Aliquot Samples for Qubit/Bioanalyzer" : {
+            "lims_instrument" : {
+                "dest_file" : "Bravo",
+                "details" : []
+            }
+        },
+        "Aliquot Libraries for Hybridization (SS XT)" : {
+            "lims_instrument" : {
+                "dest_file" : "Bravo",
+                "details" : []
+            },
+            "default" : {
+                "dest_file" : "Speedvac",
+                "details" : []
+            }
+        },
+        "Amplify Adapter-Ligated Library (SS XT) 4.0" : {
+            "lims_instrument" : {
+                "dest_file" : "Bravo",
+                "details" : []
+            },
+            "udf_PCR Machine" : {
+                "dest_file" : "PCR",
+                "details" : []
+            }
+        },
+        "Amplify Captured Libraries to Add Index Tags (SS XT) 4.0" : {
+            "lims_instrument" : {
+                "dest_file" : "Bravo",
+                "details" : []
+            },
+            "udf_PCR Machine" : {
+                "dest_file" : "PCR",
+                "details" : []
+            }
+        },
+        "Amplify by PCR and Add Index Tags (TruSeq small RNA) 1.0" : {
+            "lims_instrument" : {
+                "dest_file" : "PCR",
+                "details" : []
+            }
+        },
+        "Automated Quant-iT QC (DNA) 4.0" : {
+            "lims_instrument" : {
+                "dest_file" : "Bravo",
+                "details" : []
+            },
+            "default" : {
+                "dest_file" : "Tecan",
+                "details" : {
```

Comment:
Doesn't have to be a `dict` if the value is always empty, it can be a list (unless you are going to modify this later and add some value to it actually). I also noticed that downstream in the `get_details` method you always take the string only from its 4th index. If that is the case always then maybe you can save it as such i.e. without the prefix "udf_"

```
"default" : {
      "dest_file" : "Tecan",
      "details" : ["Assay type", "Lot no: Quant-iT reagent kit"]
 }
```